### PR TITLE
Improved description of grabService

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Explaining test.service_container

### CODE REVIEW

File: src/Codeception/Module/Symfony.php [hunk 1]
```diff
@@ -554,8 +554,9 @@ public function dontSeeEmailIsSent()
     }
 
     /**
-     * Grabs a service from Symfony DIC container.
-     * Recommended to use for unit testing.
+     * Grabs a service from the Symfony container.
```
Comment (c33s) [inline src/Codeception/Module/Symfony.php:557]: please leave DIC in the text

**Review by c33s — CHANGES_REQUESTED** [review]

File: src/Codeception/Module/Symfony.php (hunk 1 above)
Comment (ThomasLandauer) [reply src/Codeception/Module/Symfony.php:557]: "dependency injection container"?

Comment (c33s) [reply src/Codeception/Module/Symfony.php:557]: yes. sometimes you may search for DIC and then you will find this section :)

Comment (ThomasLandauer) [reply src/Codeception/Module/Symfony.php]: Well, OK, but not even the Symfony Documentation finds anything when you search for "DIC" ;-) https://symfony.com/doc/current/index.html

**Pull request merged**